src/book/02-system/01-foundations/01-overview.mdx

```diff
@@ -22,3 +22,42 @@ After motions is hardware IO. The hardware will output the commands and input th
+| `headbehtest`        | Real robot sensors and camera, vision modules, script modules, head controller and behaviour          | Used to test head behaviour, particularly with balls and goals.                                                                                                                                                                                              |
+| `keyboardwalk`       | Real robot sensors, motions, keyboard walk and NUsight                                                | Used for demonstrating the robot manually with a keyboard, particularly in a setting without a field.                                                                                                                                                                   |
+| `keyboardwalkfake`   | As above but with a basic simulated robot (not webots)                                                | Used for testing the code without a real robot, potentially to make sure there are no runtime errors. Used to demonstrate basic functionality of the robot. Part of the [Getting Started](/guides/main/getting-started) example since it does not require a real robot. |
+| `localisationtest`   | Localisation, localisation simulator and NUsight                                                      | Used to test localisation in a simulated environment.                                                                                                                                                                                                                   |
+| `lookatball`         | Real robot sensors, vision and motions                                                                | Used to test head behaviour where just the ball is the focus. By removing robot localisation, just the ball should be looked at.                                                                                                                                        |
+| `lookatgoal`         | Real robot sensors, localisation, vision and motions                                                  | Used to test head behaviour with localisation, which should make it look at the goals.                                                                                                                                                                                  |
+| `natnet`             | Real robot sensors, NUsight and NatNet                                                                | Used to get motion capture data from a [Motive](https://optitrack.com/software/motive/) motion capture system.                                                                                                                                                          |
```

Comment:
glossary addition? :heart_eyes:

Reply:
Added, although can someone make sure my wording is ok for the NatNet entry? @Bidski?